std/math.d

```diff
@@ -4808,9 +4808,10 @@ unittest
     }
     ensureDefaults();
 
+    version(D_HardFloat)
     {
-        FloatingPointControl ctrl;
-        ctrl.rounding = FloatingPointControl.roundDown;
+        FloatingPointControl ctrlr;
+        ctrlr.rounding = FloatingPointControl.roundDown;
```

Comment:
Hmm, I had to rename `ctrl` here because the OS X auto-tester said the `ctrl` below was shadowed once I added the version check, which makes sense except linux and FreeBSD both passed the auto-tester.  Bug in the compiler, or the OS X flags on the auto-tester are more stringent?